src/utils/metadata.ts

```diff
@@ -10,6 +10,66 @@ export function transformTags(value: string): string[] {
   return transformedTags
 }
 
+export function mapTimeoutStringToSeconds(timeout: string): number {
+  switch (timeout) {
+    case 'Forever':
+      return 0
+    case '1 day':
+      return 86400
+    case '1 week':
+      return 604800
+    case '1 month':
+      return 2630000
+    case '1 year':
+      return 31556952
+    default:
+      return 0
+  }
+}
+
+function numberEnding(number: number): string {
+  return number > 1 ? 's' : ''
+}
+
+export function millisecondsToStr(milliseconds: number): string {
```

Comment:
while trying to make this method a bit shorter, realized we should create a method which does exactly what we need. And we need seconds → string. So would suggest making this into `secondsToString()` so we do not need that other `mapSecondsToTimeoutString()` method too. If milliseconds are then required for internal calculation, this can happen within the method

Reply:
I can take care of that.